src/test/java/it/pagopa/pn/cucumber/steps/pa/AvanzamentoNotificheB2bSteps.java

```diff
@@ -3237,5 +3273,4 @@ public void verificaCostoePesoInvioCartaceo(Integer costo, Integer peso){
     }
     }
 
-
```

Comment:
Anche se ininfluenti per evitare conflitti revertare le modifiche di import in questa classe

Reply:
modifica effettuata come da richiesta